app/Filament/Server/Pages/Settings.php

```diff
@@ -99,21 +100,52 @@ public function form(Form $form): Form
                                 'lg' => 3,
                             ])
                             ->schema([
+                                TextInput::make('cpu')
+                                    ->label('')
+                                    ->prefix('CPU')
+                                    ->prefixIcon('tabler-cpu')
+                                    ->columnSpan(1)
+                                    ->disabled()
+                                    ->formatStateUsing(fn ($state, Server $server) => !$state ? 'Unlimited' : Number::format($server->cpu, locale: auth()->user()->language) . '%'),
+                                TextInput::make('memory')
+                                    ->label('')
+                                    ->prefix('Memory')
+                                    ->prefixIcon('tabler-device-desktop-analytics')
+                                    ->columnSpan(1)
+                                    ->disabled()
+                                    ->formatStateUsing(fn ($state, Server $server) => !$state ? 'Unlimited' : (config('panel.use_binary_prefix')
+                                        ? Number::format($server->memory / 1024, maxPrecision: 2, locale: auth()->user()->language) .' GiB'
+                                        : Number::format($server->memory / 1000, maxPrecision: 2, locale: auth()->user()->language) . ' GB')),
```

Comment:
Simplify this by using this helper method: `convert_bytes_to_readable`

https://github.com/pelican-dev/panel/blob/9d48799c2819ad7a53508273d1cf651ccfed73ad/app/helpers.php#L21-L36

Reply:
This is not a valid option, as this isn't usage, and is a set amount